README.md

```diff
@@ -33,33 +37,59 @@ CARGO_TARGET_DIR=target cargo build --release --target wasm32-unknown-unknown
+### 3. Package your Wasms into a DNA file
+
+1. Create a new dna workdir with `hc dna init <DNA_FOLDER>`.
+  - This will create a `dna.yaml` in it with the necessary initial configuration.
+2. Add your zomes to the `dna.yaml` file with references the `*.wasm` files you built in the previous step (see `workdir/dna/dna.yaml` for examples).
+3. Run the following command to package your Wasms into a DNA file per your `dna.yaml`:
+
+```bash
+hc dna pack workdir/dna
+```
+
+This will produce a `demo.dna` file as a sibling of the `workdir/dna` directory.
+
+### 4. Package your DNAs into a happ file
 
-*Note: Soon, this process will be easier in that it will not require a `.dna.workdir`*
+_hApps_ (holochain apps) are bundled as aggregations of different DNAs.
 
-1. Create a `demo.dna.workdir` directory (replace "demo" with whatever you want)
-2. Create a `demo.dna.workdir/dna.json` file which references the `*.wasm` files you built in the previous step. See the [dna.json](dna.json) file in this repo for an example.
-  - Note: this is a bit hacky right now. Normally when using a dna.workdir, you would include the Wasms alongside the `dna.json` in the same directory. However, it is easier for the purposes of this tutorial to let the `dna.json` reference Wasms in a different directory. The workdir construct becomes more useful when you need to go back and forth between an already-built DNA and its constituent Wasms.
-3. Run the following command to assemble your Wasms into a DNA file per your dna.json:
+1. Create a new happ workdir with `hc app init <HAPP_FOLDER>`.
+  - This will create a `happ.yaml` in it with the necessary initial configuration.
+2. Add the DNA bundle created in the previous step to the new `happ.yaml` file (see `workdir/happ/happ.yaml` for an example).
+3. Run the following command to package your DNAs into a happ bundle per your `happ.yaml`:
 
 ```bash
-dna-util -c demo.dna.workdir
+hc app pack workdir/happ
 ```
 
-This will produce a `demo.dna.gz` file as a sibling of the `demo.dna.workdir` directory.
+This will produce a `demo-happ.happ` file as a sibling of the `workdir/happ` directory.
 
-### 4. Use the Conductor's admin interface to install your DNA
+### 5. Running and testing your happ
 
-If you are using Tryorama to run tests against your DNA, you can jump over to the [tryorama README](https://github.com/holochain/tryorama-rsm) and follow the instructions there.
+If you are using Tryorama to run tests against your DNA, you can jump over to the [tryorama README](https://github.com/holochain/tryorama) and follow the instructions there.
 
-If you are running Holochain using your own setup, you'll have to have a deeper understanding of Holochain than is in scope for this tutorial. Roughly speaking, you'll need to:
+If you want to run the happ bundle directly, run this replacing `workdir/happ` for the directory in which you have you `*.happ` file:
+
+```bash
+hc sandbox generate workdir/happ --run
+```
```

Comment:
Might be worth mentioning that `sandbox` is still a prototype and `generate` is experimental. They are both subject to change .
I also wonder how useful this example is given that there's no UI because once you generate and run then you can't interact with your zome.

Reply:
Yeah I think it is important just because people expect to be able to run it somehow. I think I'll note it's a prototype, and if we receive feedback about the no UI thing we'll reconsider.